### ISSUE TITLE
The "K" suffix is not localised

### ISSUE THREAD

**Issue by agharbeia** [hatnote/top#37]
In the statistics line of each entry in the index, the suffix "K" of the number of views is not localised.
Both Arabic and Farsi are affected:
![wikipedia-top-100-k-suffix-localisation-bug](https://user-images.githubusercontent.com/2234596/28334406-4eca2ecc-6bfb-11e7-8baa-3b45131db21f.png)

In Arabic it should be "ك"

**Comment by mahmoud** [comment]
Hehe, you're really going over this with a fine-tooth comb! In this case however, I don't think this suffix is necessarily localization-worthy. For one, it's been my experience in Iran that "K" is pretty well understood (e.g., km for distance). It's also an [international standard](https://en.wikipedia.org/wiki/Metric_prefix). It's basically part of the lingua franca that one can reasonably expect our audience to grasp. I'm more concerned that the numbers in those screenshots appear not to be localized!

**Comment by agharbeia** [comment]
Well, ك is used in Arabic in physics and mathematics. And being a standard is not a reason not to localise it ;)
Also the "K" causes BiDi problems, both in Arabic and Farsi, specially when tweeting.
It would be great to solve if possible.

As for the numerals, I had written a issue about the opposite situation: In Chromium the numerals looked enforcedly localised to the eastern variety of Arabic numerals while my preference is the western. I hesitantly deduced that you must be hard-coding the unicode points for Arabic and Farsi to U+0660 to U+0669 and the U+06F0 to U+06F9 ranges, which is not a good thing in my opinion because it doesn't leave the choice to the reader/user.

But it turns out that I had set Firefox to always display the western variety of Arabic numerals but not Chromium. So I didn't submit that issue. This screenshot is from Firefox.

So in short: leave the numerals as is. They're fine. Users can choose in their client application. Arabic readers are almost equally divided in their preference for numerals. It's a historical Maghrib vs. Mashriq thing.

**Comment by agharbeia** [comment]
Now that I fetched the page using wget and looked at the source, you are indeed hard-coding the numerals. I really recommend that you don't. This should be handled by user preferences in the client.
It will look funny in some clients when some numerals are displayed in one variety and the other in the other variety in the same page.
